apollo-router/src/testdata/supergraph.graphql

```diff
@@ -68,6 +68,7 @@ enum link__Purpose {
 
 type Product
   @join__type(graph: INVENTORY, key: "upc")
+  @join__type(graph: INVENTORY, key: "upc inStock")
```

Comment:
Can we put this in a separate file so other tests don't also have this extra key? It could just literally be a copy of this but specific for your new test. Since existing tests are passing it probably doesn't have a negative effect but I think putting things highly specific for one test in this supergraph file that's shared by loads of tests is going to be risky going forward

Reply:
Done